geometry_msgs/PolygonStamped inputPolygon
---
geometry_msgs/PolygonStamped[] outputPolygons